pgactivity/queries/get_blocking.sql

```diff
@@ -125,18 +131,19 @@ SELECT
                 ELSE extract(epoch from now() - {duration_column}) > %(min_duration)s
             END
       ) AS sq
-GROUP BY
-      pid,
-      application_name,
-      query,
-      mode,
-      locktype,
-      duration,
-      datname,
-      usename,
-      client,
-      state,
-      relation,
-      wait
+      LEFT OUTER JOIN pg_database b ON sq.datid = b.oid
+--GROUP BY
+--      pid,
+--      application_name,
+--      query,
+--      mode,
+--      locktype,
+--      duration,
+--      database,
+--      usename,
+--      client,
+--      state,
+--      relation,
+--      wait
```

Comment:
Should we keep these commented lines?

Reply:
According to the [pg_locks docs](https://www.postgresql.org/docs/14/view-pg-locks.html)
> A single process can be waiting to acquire at most one lock at a time.

Therefore: 

* the blocking => blocked join can return only one row (at most). 
* blocked => pg_stat_activity will return one row only

The `group by` seems useless to me.

I will remove the block as suggested.

Reply:
According to the [pg_locks docs](https://www.postgresql.org/docs/14/view-pg-locks.html)
> A single process can be waiting to acquire at most one lock at a time.

Therefore: 

* the blocking => blocked join can return several rows. 
* blocked => pg_stat_activity will return one row only

It's a mistake. I will put it back and amend the test.

Reply:
Fixed the `group by` thing  in f5818a4